lte/protos/sms_orc8r.proto

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+import "orc8r/protos/common.proto";
+
+package magma.lte;
+option go_package = "magma/lte/cloud/go/protos";
+
+service SMSOrc8rService {
+    rpc SMOUplink (SMOUplinkUnitdata) returns (magma.orc8r.Void) {}
+}
+
+service SMSOrc8rGatewayService {
+    rpc SMODownlink (SMODownlinkUnitdata) returns (magma.orc8r.Void) {}
+    rpc SMOPagingReq (SMOPagingRequest) returns (magma.orc8r.Void) {}
```

Comment:
why do we need a separate RPC for paging? Should not SMODownlink rpc call trigger paging and message delivery?